app/src/main/java/de/westnordost/streetcomplete/quests/drinking_water/AddDrinkingWater.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.drinking_water
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+
+class AddDrinkingWater : OsmFilterQuestType<DrinkingWater>() {
+
+    override val elementFilter = """
+        nodes with (
+            man_made = water_tap
+            or man_made = water_well
+            or amenity = toilets
+            or natural = spring
+        )
+        and access !~ private|no and indoor != yes
+        and !drinking_water and amenity != drinking_water
+    """
+
+    override val commitMessage = "Add whether water is drinkable"
+    override val wikiLink = "Key:drinking_water"
+    override val icon = R.drawable.ic_quest_drinking_water
+    override val isDeleteElementEnabled = true
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_drinking_water_title
```

Comment:
I think it makes sense to have a different question for the different things to ask for. I.e. the user should know if (s)he is looking for a faucet, or a natural spring, or a water well.

However, we could try something different here.

We have the feature dictionary. If all of man_made = water_tap, man_made = water_well etc. are in the id presets, the question could also appear like this:

`Is the water drinkable here? (Natural spring)`

This helps to keep down the number of strings that must be translated.

Reply:
Whether a preset for a thing exists, can be checked by searching through the `presets.json` (not the brands one)

Reply:
They are indeed all included in `presets.json`:

* [`man_made=water_tap`](https://github.com/openstreetmap/id-tagging-schema/blob/9fdecc983443ce75185c2923caae9f7703f2cae4/dist/presets.json#L15735)
* [`man_made=water_well`](https://github.com/openstreetmap/id-tagging-schema/blob/9fdecc983443ce75185c2923caae9f7703f2cae4/dist/presets.json#L15768)
* [`natural=spring`](https://github.com/openstreetmap/id-tagging-schema/blob/9fdecc983443ce75185c2923caae9f7703f2cae4/dist/presets.json#L16297)

How can I access those from the code?

Reply:
See for example `AddPlaceName.kt`, `getTitle` and `getTitleArgs`

Reply:
I think `arrayOf(featureName.value.toString())` should be fine too. In the worst case (-> somehow that preset was removed), the question would appear as `Is the water drinkable here? (null)`